### PR TITLE
Fix content in script editor

### PR DESCRIPTION
**Pull request by zaelgohary**
### Description

- Fix content in provider

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1186

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/UI/examples/html-tag-example/index.html [hunk 1]
```diff
@@ -7,17 +7,20 @@
     <title>Threefold UI</title>
 
     <!-- Include the PDF style component, uncomment the script line after copying the build js/css in the same dir -->
-    <!-- <link rel="stylesheet" href="./style.css"> -->
+    <link rel="stylesheet" href="./tailwind.css">
```
Comment (Mahmoud-Emad) [inline packages/UI/examples/html-tag-example/index.html:10]: What do you think if we added some dummy files in the same dir, the `tailwind.css` and  `threefold-ui.umd.js` with no content and one single comment in each file to tell the user that he just needs to copy the build and past them here in these files?

File: packages/UI/examples/html-tag-example/index.html [hunk 2]
```diff
@@ -7,17 +7,20 @@
...
-    <pdf-signer dest="http://localhost:3000/"
+    <pdf-signer dest="http://localhost:3000/api/verify"
         pdfUrl="https://raw.githubusercontent.com/mozilla/pdf.js/ba2edeae/web/compressed.tracemonkey-pldi-09.pdf"
         network="dev" />
+
+        <!-- <script-editor dest="http://localhost:3000/api/verify"
+        network="dev" /> -->
     <!-- Include the PDF Signer component script, uncomment the script line after copying the build js/css in the same dir -->
-    <!-- <script src="./pdf.js"></script>  -->
+    <script src="./threefold-ui.umd.js"></script>
```
Comment (Mahmoud-Emad) [inline packages/UI/examples/html-tag-example/index.html:23]: Same goes here

**Review by Mahmoud-Emad — COMMENTED** [review]
Great work, I just added one single comment that might enhance the user's understanding and readability.

File: packages/UI/examples/html-tag-example/index.html (hunk 1 above)
Comment (zaelgohary) [reply packages/UI/examples/html-tag-example/index.html:10]: This has been already explained in the docs [here](https://github.com/threefoldtech/tfgrid-sdk-ts/blob/development/packages/UI/docs/script_editor.md#building-the-script-editor).

Comment (Mahmoud-Emad) [reply packages/UI/examples/html-tag-example/index.html:10]: Well done

**Review by Mahmoud-Emad — APPROVED** [review]

**Pull request merged**